src/libraries/shims/System.Threading.Timer/ref/System.Threading.Timer.cs

```diff
@@ -1,8 +1,5 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-// ------------------------------------------------------------------------------
-// Changes to this file must follow the https://aka.ms/api-review process.
-// ------------------------------------------------------------------------------
```

Comment:
I thought this was an automated comment?

Reply:
We have an editorconfig rule that forces the two line license header in code files under a src folder. As this file is included by the src\System.Threading.Timer.csproj project, the header had to be shortened. Also, as the file now only contains type forwards, changes don't need to through api-review process anymore.